### PR TITLE
Doc updates incorporating Isaac's feedback

### PR DESCRIPTION
**Pull request by waltergoulet-okta**
Updates from Isaac review here: https://docs.google.com/document/d/1Eqyt_SSNvQQEV5i4Ki_6iiIt99Lc_Fn1aml5Ehta0GI/edit?usp=sharing

### CODE REVIEW

File: oktapam/constants/descriptions/attributes.go [hunk 1]
```diff
@@ -33,25 +33,25 @@ var (
...
 	ClusterSelector                  = "A label selector to used to match Kubernetes clusters."
-	CreateServerGroup                = "If 'true', `sftd` (ASA Server Agent) creates a corresponding local (unix or windows) group in the ASA Project's servers."
-	CreateServerUsers                = "If 'true', `sftd` (ASA Server Agent) creates corresponding local (unix or windows) user accounts in the ASA Project's servers."
-	CreatedAt                        = "The UTC time of resource deletion. Format is `2022-01-01 00:00:00 +0000 UTC`."
+	CreateServerGroup                = "If `true`, 'sftd' (ASA Server Agent) creates a corresponding local (unix or windows) group in the ASA Project's servers."
+	CreateServerUsers                = "If `true`, 'sftd' (ASA Server Agent) creates corresponding local (unix or windows) user accounts in the ASA Project's servers."
+	CreatedAt                        = "The UTC time when the resource was created. Format is '2022-01-01 00:00:00 +0000 UTC'."
```
Comment (stevenelleman-okta) [inline oktapam/constants/descriptions/attributes.go:38]: Good catch.

File: oktapam/constants/descriptions/attributes.go [hunk 2]
```diff
@@ -33,25 +33,25 @@ var (
...
 	CreatedByUser                    = "The ASA User that created the resource."
 	CSRDetails                       = "Certificate Signing Request (CSR) Details."
 	DefaultAddress                   = "Default address of the gateway."
-	DeletedAt                        = "The UTC time of resource creation. Format is `2022-01-01 00:00:00 +0000 UTC`."
+	DeletedAt                        = "The UTC time of resource creation. Format is '2022-01-01 00:00:00 +0000 UTC'."
```
Comment (stevenelleman-okta) [inline oktapam/constants/descriptions/attributes.go:42]: *resource deletion, whoops.

File: oktapam/constants/descriptions/attributes.go [hunk 3]
```diff
@@ -65,24 +65,24 @@ var (
...
-		"Otherwise, please use a more current key algorithm. If left unspecified, `%s` is used by default.", typed_strings.SSHCertTypeListFormat(), typed_strings.CertTypeRsa, typed_strings.CertTypeRsa, typed_strings.CertTypeEd25519)
-	SSHSessionRecording = "If 'true', enables ssh recording on server access requests."
-	Status              = "The status of the ASA User. Valid statuses are `ACTIVE`, `DISABLED`, and `DELETED`."
+	SSHCertificateType               = fmt.Sprintf("The SSH certificate type used by access requests. Options include: [%s]. '%s' is a deprecated key algorithm type. "+
+		"This option should only be used to connect to legacy systems that cannot use newer SSH versions. If you do need to use '%s', it is recommended to connect via a gateway with traffic forwarding. "+
+		"Otherwise, please use a more current key algorithm. If left unspecified, '%s' is used by default.", typed_strings.SSHCertTypeListFormat(), typed_strings.CertTypeRsa, typed_strings.CertTypeRsa, typed_strings.CertTypeEd25519)
```
Comment (stevenelleman-okta) [inline oktapam/constants/descriptions/attributes.go:81]: If we're changing the cert types to be wrapped in single question, this `ListFormat()` method will need to be changed to reflect that as well.

File: oktapam/constants/descriptions/attributes.go [hunk 4]
```diff
@@ -65,24 +65,24 @@ var (
...
+	SSHSessionRecording = "If `true`, enables ssh recording on server access requests."
+	Status              = "The status of the ASA User. Valid statuses are 'ACTIVE', 'DISABLED', and 'DELETED'."
```
Comment (stevenelleman-okta) [inline oktapam/constants/descriptions/attributes.go:83]: If true uses `` shouldn't the status values also use it? Same thing with user type.

File: oktapam/constants/descriptions/data_sources.go [hunk 5]
```diff
@@ -6,24 +6,24 @@ var (
 	// Supported Data Sources
 
 	// Fetch
-	SourceGatewaySetupToken     = fmt.Sprintf("Returns a previously created ASA Gateway Setup Token. %s", LinkGatewaySetupToken)
-	SourceServerEnrollmentToken = fmt.Sprintf("Returns a previously created ASA Server Enrollment Token, corresponding to an ASA Project. %s", LinkServerEnrollmentToken)
+	SourceGatewaySetupToken     = fmt.Sprintf("Returns a previously created ASA Gateway Setup Token associated with the ASA Team specified in the OKTAPAM_TEAM environment variable. %s", LinkGatewaySetupToken)
```
Comment (stevenelleman-okta) [inline oktapam/constants/descriptions/data_sources.go:9]: Can the environment variable be passed in as a variable?

File: oktapam/constants/descriptions/data_sources.go [hunk 6]
```diff
@@ -6,24 +6,24 @@ var (
...
+	SourceServerEnrollmentToken = fmt.Sprintf("Returns a previously created ASA Server Enrollment Token associated with a specifica ASA Project. %s", LinkServerEnrollmentToken)
```
Comment (stevenelleman-okta) [inline oktapam/constants/descriptions/data_sources.go:10]: Nit: "specific"

File: oktapam/constants/descriptions/links.go [hunk 7]
```diff
@@ -5,11 +5,11 @@ const (
...
+	LinkGateway               = "For details, see [Advanced Access Server Gateways](https://help.okta.com/asa/en-us/Content/Topics/Adv_Server_Access/docs/gateways-and-bastions.htm) on ASA Gateways and bastions."
+	LinkGatewaySetupToken     = "For details, see [Configure an Advanced Server Access gateway](https://help.okta.com/asa/en-us/Content/Topics/Adv_Server_Access/docs/gateway-configure.htm)."
+	LinkGroup                 = "For details, see [Groups](https://help.okta.com/asa/en-us/Content/Topics/Adv_Server_Access/docs/setup/groups.htm)."
+	LinkProject               = "For details, [Projects](https://help.okta.com/asa/en-us/Content/Topics/Adv_Server_Access/docs/setup/projects.htm)."
+	LinkProjectGroup          = "For details, see [Add a Group to a Project](https://help.okta.com/asa/en-us/Content/Topics/Adv_Server_Access/docs/setup/add-a-group-to-project.htm)."
+	LinkServerEnrollmentToken = "For details, see [Enroll a server](https://help.okta.com/asa/en-us/Content/Topics/Adv_Server_Access/docs/setup/enrolling-a-server.htm)."
```
Comment (stevenelleman-okta) [inline oktapam/constants/descriptions/links.go:13]: Nice.

File: oktapam/constants/descriptions/data_sources.go (hunk 6 above)
Comment (waltergoulet-okta) [reply oktapam/constants/descriptions/data_sources.go:10]: Fixed

File: oktapam/constants/descriptions/attributes.go (hunk 4 above)
Comment (waltergoulet-okta) [reply oktapam/constants/descriptions/attributes.go:83]: Isaac specifically called out only the boolean fields as requiring backticks in his review to be consistent with doc team patterns, so following that paradigm.

**Review by stevenelleman-okta — APPROVED** [review]

**Pull request merged**